samples/tool/SFSClientTool.cpp

```diff
@@ -30,7 +30,7 @@ void DisplayUsage()
         << "Options:" << std::endl
         << "  -h, --help\t\t\tDisplay this help message" << std::endl
         << "  -v, --version\t\t\tDisplay the library version" << std::endl
-        << "  --isApp\t\tIndicates the specific product is an App" << std::endl
+        << "  --isApp\t\t\tIndicates the specific product is an App" << std::endl

```

Comment:
tabs always seem iffy to me.  Maybe these \t should be changed to an appropriate number of spaces?

Reply:
Hmm, I guess both would work, but that could be another task.

Reply:
Seems like overkill to create a new task just to change \t -> spaces, but ultimately your call.

Reply:
I found std::setw can help format the text. Do you think it's a better way?

    std::cout << std::left << std::setw(30) << "--product <identifier>" << "Name or GUID of the product to be retrieved\n";
    std::cout << std::left << std::setw(30) << "-h, --help" << "Display this help message\n";

Reply:
I'm not offended by that suggestion :-)

Reply:
Ok. I will complete this PR first. And will create another PR to do this change (in both SFSClientTool.cpp IntegrationDOClient.cpp).

Reply:
I'm not familiar with that. I think spaces should be enough though